core/src/main/scala/org/apache/spark/SparkEnv.scala

```diff
@@ -350,6 +356,13 @@ object SparkEnv extends Logging {
         "levels using the RDD.persist() method instead.")
     }
 
+    val outputCommitCoordinator = mockOutputCommitCoordinator.getOrElse {
+      new OutputCommitCoordinator(conf)
+    }
```

Comment:
Actually, per offline discussion with @andrewor14, I'm going to leave this as-is for now.